pootle/apps/pootle_app/project_tree.py

```diff
@@ -25,23 +25,40 @@
 LANGCODE_RE = re.compile('^[a-z]{2,3}([_-][a-z]{2,3})?(@[a-z0-9]+)?$',
                          re.IGNORECASE)
 #: Case insensitive match for language codes as postfix
+
+# this is regex for LO
+#LANGCODE_POSTFIX_RE = re.compile('^.*?[-_.]([a-z]{2,3}[-_@][a-z0-9]+|[a-z]{2,3})$', re.IGNORECASE)
+
 LANGCODE_POSTFIX_RE = re.compile(
     '^.*?[-_.]([a-z]{2,3}([_-][a-z]{2,3})?(@[a-z0-9]+)?)$', re.IGNORECASE)
+LANGCODE_SEPARATOR_RE = re.compile('([-_])')
 
 
 def direct_language_match_filename(language_code, path_name):
```

Comment:
Despite it is unrelated, I would add a docstring explaining the purpose of this function.